calculator.js

```diff
@@ -1,2 +1,27 @@
+
 var calculator = {
+  add: function (num1, num2){
+    return num1 + num2;
+  }
+
+  sub: function (num1, num2){
+    return num1 - num2;
+  }
+
+  div: function (num1, num2){
+    return num1 / num2;
+  }
+
+  mul: function (num1, num2){
+    return num1 * num2;
+  }
+ exp: function (num1, pow){
+   Math.pow(num1, pow);
```

Comment:
We need to have a `return` here or the output to `calculator.exp()` will always be `undefined`